be/src/exec/line_reader.h

```diff
@@ -28,7 +28,8 @@ class LineReader {
 public:
     virtual ~LineReader() = default;
     virtual Status read_line(const uint8_t** ptr, size_t* size, bool* eof,
-                             const io::IOContext* io_ctx) = 0;
+                             const io::IOContext* io_ctx, size_t* read_bytes = nullptr) = 0;
```

Comment:
I think we should first refactor broker load and use insert into select to implement broker load. And then we add scan bytes to all scan node . In this way, we will add progress feature to all load types, like stream load, broker load, tvf load, insert into select